ticket_to_csv.py

```diff
@@ -0,0 +1,109 @@
+import json
+import csv
+import argparse
+import pathlib
+
+
+# text preprocessing pipeline
+def preprocess_text(text):
+    text = text.replace(",", "")  # remove commas
+    text = text.replace('"', "")  # remove double quotes
+    return text
+
+# write a ticket as a row in the CSV
+def write_ticket(csv_writer, ticket, taggers):
+    ticket_id = ticket["ticket_id"]
+
+    ticket_title = preprocess_text(ticket["title"])
+    ticket_content = preprocess_text(ticket["content"])
+
+    # CSV writer takes a list as a row
+    ticket_list = [ticket_id, ticket_title, ticket_content]
+
+    # get tags for each tagger
+    tag_dict = ticket["tags"]
+    ticket_taggers = tag_dict.keys()
+    for tagger in taggers:
+        if tagger in ticket_taggers:
+            tags = "|".join(tag_dict[tagger])   # using | as a delimeter for now
```

Comment:
Will this result in a leading pipe character before the first tag in this field?
Please verify correctness against expected behaviour Maybe write a small test case exercising this code with builtin examples of tag sets.

Reply:
The join function concatenates the strings in a list with the join character, so there won't be leading or trailing delimeter characters